_posts/2015-07-21-the-epages-selenium-framework.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: "The ePages Selenium Framework"
+date: "2015-07-21 17:06:00"
+icon: ship
```

Comment:
Does the "ship" make sense here or is this just copied from another post? It would be cool, to have something related with the blog article.